### PR TITLE
{lib}[GCCcore/10.3.0,GCCcore/11.2.0] aiohttp v3.8.1 w/ Python 3.9.5 + 3.9.6

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node306.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/512bd77020f6dbc498bbe23ae059ef84 for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node305.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b4ea5d827d60001b542b8291618a2b1d for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Review by ocaisa — APPROVED** [review]
Both LGTM

File: easybuild/easyconfigs/a/aiohttp/aiohttp-3.8.1-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('typing-extensions', '3.10.0.0'),
```
Comment (ocaisa) [inline easybuild/easyconfigs/a/aiohttp/aiohttp-3.8.1-GCCcore-10.3.0.eb:17]: Just to confirm, this is a required dep? (Perhaps due to the additonal extension `charset-normalizer`)

Comment (lexming) [reply easybuild/easyconfigs/a/aiohttp/aiohttp-3.8.1-GCCcore-10.3.0.eb:17]: It's actually a direct dependency for aiohttp. See https://github.com/aio-libs/aiohttp/blob/1dbfbb433f3f7cd05ca97288805dec9e296d75de/setup.cfg#L55

Comment (lexming) [reply easybuild/easyconfigs/a/aiohttp/aiohttp-3.8.1-GCCcore-10.3.0.eb:17]: In the easyconfig for 2021b, this one moved into the main Python ec

Comment (ocaisa) [reply easybuild/easyconfigs/a/aiohttp/aiohttp-3.8.1-GCCcore-10.3.0.eb:17]: Great, thanks!

**Pull request merged**